tests/dragonfly/conftest.py

```diff
@@ -225,3 +226,40 @@ def port_picker():
 @pytest.fixture(scope="class")
 def memcached_connection(df_server: DflyInstance):
     return pymemcache.Client(f"localhost:{df_server.mc_port}")
+
+
+@pytest.fixture(scope="function")
+def tls_server_key_file_name():
+    return "df-key.pem"
+
+@pytest.fixture(scope="function")
+def tls_server_cert_file_name():
+    return "df-cert.pem"
+
+@pytest.fixture(scope="function")
```

Comment:
I think its better to use scope="session" so this function will run only once when we are running regression tests